src/box/tuple.h

```diff
@@ -170,6 +170,29 @@ box_tuple_format(box_tuple_t *tuple);
 const char *
 box_tuple_field(box_tuple_t *tuple, uint32_t fieldno);
 
+/**
+ * Return a raw tuple field in the MsgPack format pointed by
+ * a JSON path.
+ *
+ * The JSON path includes the outmost field. For example, "c" in
+ * ["a", ["b", "c"], "d"] can be accessed using "[2][2]" path (if
+ * index_base is 1, as in Lua). If index_base is set to 0, the
+ * same field will be pointed by the "[1][1]" path.
+ *
+ * The first JSON path token may be a field name if the tuple
+ * has associated format with named fields. A field of a nested
+ * map can be accessed in the same way: "foo.bar" or ".foo.bar".
+ *
+ * The buffer is valid until a next call to a box_tuple_*
+ * function.
```

Comment:
What buffer? Anyway, it looks like the return value is valid until the tuple is destroyed.

Reply:
I clarified it this way for `box_tuple_field_by_path()`, but left `box_tuple_field()` description as is for now.